src/components/Network/Network.vue

```diff
@@ -104,32 +132,106 @@ export default {
       ],
       ingresses_headers: [
         {
-          text: 'Name',
+          text: 'Hosts',
+          value: 'hosts',
           algin: 'start',
-          sortable: false,
+          sortable: true
+        },
+        {
+          text: 'Status',
+          value: 'status'
+        },
+        {
+          text: 'Name',
           value: 'name'
         },
+        {
+          text: 'Annotations',
+          value: 'annotations'
+        },
         {
           text: 'TLS',
           value: 'tls'
         },
         {
-          text: 'Hosts',
-          value: 'hosts'
+          text: 'Paths',
+          value: 'path'
+        },
+        {
+          text: 'Service',
+          value: 'services'
+        },
+        {
+          text: 'Created at',
+          value: 'creation_timestamp'
         }
       ]
     }
   },
-  mounted () {
+  beforeMount () {
     if (this.services_data.length === 0) {
       this.$store.dispatch('services/request_services')
     }
     if (this.ingresses_data.length === 0) {
       this.$store.dispatch('ingresses/request_ingresses')
     }
+  },
+  mounted () {
+    this.check_available()
+  },
+  methods: {
+    check_availablity (data) {
+      this.ingress = data
+      data['status'] = data.rules.map(async rule => {
+        console.log('test route: ' + `${rule.host}`)
+        let statuscode
+        const XHR = new XMLHttpRequest()
+        if (rule.paths.length < 2) {
+          XHR.open('GET', `${rule.host}`)
+          let loading = () => {
+            if (XHR.status === 200) {
+              console.log('################')
+              console.log(XHR.status)
+              data['status'] = 200
+            } else {
+              console.warn(XHR.statusText, XHR.responseText)
+              console.log('################')
+              console.log(XHR.status)
+              statuscode = XHR.status
+              data['status'] = XHR.status
+            }
+          }
+          XHR.addEventListener('load', loading)
+          XHR.send()
+        } else {
+          rule.paths.map(path => {
+            console.log('hhhhhhhhhhhhhhhhhhhhhh' + `${rule.host}${path.path}`)
```

Comment:
And before we merge could you remove unnecessary console.logs and add readable logic to necessary ones?